sdks/python/apache_beam/io/tfrecordio.py

```diff
@@ -0,0 +1,272 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+"""TFRecord sources and sinks."""
+
+from __future__ import absolute_import
+
+import logging
+import struct
+
+from apache_beam import coders
+from apache_beam.io import filebasedsource
+from apache_beam.io import fileio
+from apache_beam.io.iobase import Read
+from apache_beam.io.iobase import Write
+from apache_beam.transforms import PTransform
+import crcmod
+
+__all__ = ['ReadFromTFRecord', 'WriteToTFRecord']
+
+
+def _default_crc32c_fn(value):
+  """Calculates crc32c by either snappy or crcmod based on installation."""
+
+  if _default_crc32c_fn.fn:
```

Comment:
If you invert the condition here, you can have a single return statement. (I think it is fine either way.)